tests/foreman/ui/test_syncplan.py

```diff
@@ -30,15 +30,17 @@
 from robottelo.datafactory import valid_cron_expressions
 
 
-def validate_task_status(repo_id, max_tries=10):
+def validate_task_status(repo_id, module_org_id, max_tries=10):
     """Wait for foreman_tasks to complete or timeout
 
     :param repo_id: Repository Id to identify the correct task
     :param max_tries: Max tries to poll for the task creation
-        pulp tasks
+    :param organization_id: Org ID to ensure valid check on busy Satellite
```

Comment:
```suggestion
    :param module_org_id: Org ID to ensure valid check on busy Satellite
```

Reply:
Hello @jyejare , thank you for spotting that, now fixed.